app/src/main/java/com/github/se/cyrcle/ui/navigation/NavigationActions.kt

```diff
@@ -22,74 +24,44 @@ object Screen {
   const val REVIEW = "Review Screen"
   const val LOCATION_PICKER = "Location Picker"
   const val ATTRIBUTES_PICKER = "Attributes Picker"
+  const val PROFILE = "Profile Screen" // Add this
 }
 
-/**
- * Data class representing a top level destination in the app.
- *
- * @param route The route of the destination
- * @param icon The icon to display for the destination
- * @param textId The string resource ID for the text to display for the destination
- */
 data class TopLevelDestination(val route: String, val icon: ImageVector, val textId: String)
 
-/** Object containing the top level destinations in the app. */
 object TopLevelDestinations {
   val LIST =
       TopLevelDestination(route = Route.LIST, icon = Icons.Outlined.Menu, textId = Route.LIST)
   val MAP =
       TopLevelDestination(route = Route.MAP, icon = Icons.Outlined.LocationOn, textId = Route.MAP)
+  val PROFILE =
+      TopLevelDestination(
+          route = Route.PROFILE, icon = Icons.Outlined.Person, textId = Route.PROFILE)
 }
 
-/** List of top level destinations in the app. */
-val LIST_TOP_LEVEL_DESTINATION = listOf(TopLevelDestinations.MAP, TopLevelDestinations.LIST)
+val LIST_TOP_LEVEL_DESTINATION =
+    listOf(TopLevelDestinations.MAP, TopLevelDestinations.LIST, TopLevelDestinations.PROFILE)
```

Comment:
(Not very important rn) I don't think profile should be a TLD. For now we should keep it like this, we can discuss this with the rest of the team another time.

Reply:
I think it should, it is like that on every single app I use on my phone. Debatable indeed.